lib/workers/repository/onboarding/pr/index.ts

```diff
@@ -39,6 +40,25 @@ export async function ensureOnboardingPr(
   logger.trace({ config });
   // TODO #7154
   const existingPr = await platform.getBranchPr(config.onboardingBranch!);
+  if (existingPr) {
+    // skip pr-update if branch is conflicted
+    if (
+      (await scm.isBranchModified(config.onboardingBranch!)) &&
```

Comment:
Why does it matter if it's modified? If it's conflicted.. it's conflicted

Reply:
I added this fortwo reasons:
1. `isBranchModified` is stored so we don't need to compute it twice like conflicted
2. conflicted causes throws error-which is caught & logged (safe) 

If we check modified first we prevent one extra git comuptation on branches that aren't modified. And we don't have to confuse the error-message in logs.
Other than that not any important use-case

Reply:
But an unmodified branch could still be conflicted. It's unlikely.. because it would imply they have added their own repo config, which would mean onboarding is no longer necessary.

Reply:
In this case, we would detect that `repoIsOnboarded=true`during `isOnboarded()` and return before checking for modified or conflicted branc